### ISSUE TITLE
你好，非常感谢您的分享！

### ISSUE THREAD

**Issue by SongLongZhang** [DUTFangXiang/ResnetforSaliency#1]
你好，我叫张松龙，我在训练你模型的时候遇到了一些问题，想请教一下您！我训练的时候发现loss一直等于  
Train net output #2: loss = 1.08829e+07 (* 1 = 1.08829e+07 loss)
loss不下降，请问您训练的时候遇到了这样的问题吗？请问，您是怎么解决的呢？
